src-java/server42/server42-control-messaging/src/main/java/org/openkilda/server42/control/messaging/flowrtt/Control.java

```diff
@@ -41,6 +41,40 @@ public static void registerAllExtensions(
      * <code>int64 tunnel_id = 3;</code>
      */
     long getTunnelId();
+
+    /**
+     * <code>.org.openkilda.server42.control.messaging.flowrtt.Flow.EncapsulationType transit_encapsulation_type = 4;</code>
+     */
+    int getTransitEncapsulationTypeValue();
+    /**
+     * <code>.org.openkilda.server42.control.messaging.flowrtt.Flow.EncapsulationType transit_encapsulation_type = 4;</code>
+     */
+    org.openkilda.server42.control.messaging.flowrtt.Control.Flow.EncapsulationType getTransitEncapsulationType();
```

Comment:
Why do we need to know anything about **transit** encapsulation? Kilda matches RTT packets by Input vlan and as I understand we store input vlan in `getEncapsulationTypeValue` field